### PR TITLE
scrot: add page

### PR DESCRIPTION
**Pull request by reneatzi**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/scrot.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Capture a screenshot:
+
+`scrot`
+
+- Capture a screenshot interactively:
+
+`scrot --select`
+
+- Capture a screenshot from the currently focued window:
```
Comment (schneiderl) [inline pages/linux/scrot.md]: I believe it should be "focused" insted of focued.

Comment (reneatzi) [reply pages/linux/scrot.md]: Thank you, fixed it.

File: pages/linux/scrot.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`scrot`
+
+- Capture a screenshot interactively:
+
+`scrot --select`
+
+- Capture a screenshot from the currently focused window:
+
+`scrot --focused`
```
Comment (mfrw) [inline pages/linux/scrot.md]: How about adding an example with a filename.
```suggestion
`scrot --focused {{capture.png}}`
```

File: pages/linux/scrot.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Capture a screenshot:
+
+`scrot`
+
+- Capture a screenshot interactively:
+
+`scrot --select`
+
+- Capture a screenshot from the currently focused window:
```
Comment (mfrw) [inline pages/linux/scrot.md]:
```suggestion
- Capture a screenshot from the currently focused window and save it as "capture.png":
```

File: pages/linux/scrot.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Screen capture utility.
+
+- Capture a screenshot:
```
Comment (mfrw) [inline pages/linux/scrot.md]: Should we add that it dumps the screenshot as a date-stamped file in the current directory?

File: pages/linux/scrot.md (hunk 2 above)
Comment (sbrl) [reply pages/linux/scrot.md]: Perhaps we could do that in a separate example, @mfrw?

File: pages/linux/scrot.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/scrot.md]: See above.

File: pages/linux/scrot.md (hunk 4 above)
Comment (sbrl) [reply pages/linux/scrot.md]: Hrm, maybe. What about this?

```suggestion
- Capture a screenshot and save it to the current directory with the current date as the filename:
```

File: pages/linux/scrot.md (hunk 3 above)
Comment (reneatzi) [reply pages/linux/scrot.md]: Done, added a new example

**Comment by sbrl** [comment]
Can we merge this @mfrw?

**Comment by pxgamer** [comment]
Ping @mfrw, any update on this? 🙂

**Comment by mfrw** [comment]
@pxgamer yes! Sorry I missed the previous notification. Downed in notifications :(

**Comment by sbrl** [comment]
Yeah, it can be a challenge sometimes to keep on top of them all, @mfrw. Coming up with an efficient system for doing so is hard.

I've found that learning the GH keyboard shortcuts is helpful. In particular `g` then `n` is helpful to warp back to my notifications list

**Comment by mfrw** [comment]
Thank you very much for the pro tip

**Comment by sbrl** [comment]
Pressing `?` will open a list of all the shortcuts available

@mfrw

**Pull request merged**